@cci/cci@v1.ifcx

```diff
@@ -0,0 +1,27 @@
+{
+  "header": {
+    "id": "https://identifier.buildingsmart.org/uri/molio/cciconstruction/1.0",
+    "version": "ifcx_alpha",
+    "author": "authorname",
+    "timestamp": "time string"
+  },
+  "imports": [],
+  "schemas": {
+    "CCIConstruction::class": {
```

Comment:
The IFC5 could have a class definition (code+uri) already in the schema, then we wouldn't have to redefine the class object for every classification. It's fine here, I will add an issue.